src/main/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/FlagStatDataflow.java

```diff
@@ -0,0 +1,26 @@
+package org.broadinstitute.hellbender.tools.dataflow.pipelines;
+
+
+import com.google.common.collect.ImmutableList;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.programgroups.DataFlowProgramGroup;
+import org.broadinstitute.hellbender.engine.dataflow.PTransformSAM;
+import org.broadinstitute.hellbender.engine.filters.ReadFilter;
+import org.broadinstitute.hellbender.engine.filters.ReadFilterLibrary;
+import org.broadinstitute.hellbender.tools.FlagStat;
+import org.broadinstitute.hellbender.tools.dataflow.transforms.FlagStatusDataflowTransform;
+
+@CommandLineProgramProperties(usage="runs FlagStat on dataflow", usageShort = "FlagStat", programGroup = DataFlowProgramGroup.class)
+public class FlagStatDataflow extends DataflowReadsPipeline {
```

Comment:
all these little 1-tool pipelines are ok for now but will need to go at some point. it'd be insane to have a separate pipeline class per tool